sdcm/ycsb_thread.py

```diff
@@ -254,7 +254,7 @@ def raise_event_callback(sentinel, line):  # pylint: disable=unused-argument
                     log_file=log_file_name,
                     watchers=[
                         FailuresWatcher(
-                            r'\sERROR|=UNEXPECTED_STATE',
+                            r'\sERROR|=UNEXPECTED_STATE|=ERROR',
```

Comment:
do we report the type of the failure somewhere? whether it is 'unexpected' or 'error'?

Reply:
the line itself is part of the error